global-api/migrations/versions/cc8d00d4372e_.py

```diff
@@ -0,0 +1,26 @@
+"""empty message
```

Comment:
This seems to be an unused migration file. Remove it or use it to change the nullable attribute of emission_factor_value and emission_factor_units and update the documentation comments and file name.